sdk_v2/kubeflow/training/constants/constants.py

```diff
@@ -0,0 +1,128 @@
+# Copyright 2024 The Kubeflow Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+# How long to wait in seconds for requests to the Kubernetes API Server.
+DEFAULT_TIMEOUT = 120
+
+# Common constants.
+GROUP = "kubeflow.org"
+VERSION = "v2alpha1"
+API_VERSION = f"{GROUP}/{VERSION}"
+
+# The default Kubernetes namespace.
+DEFAULT_NAMESPACE = "default"
+
+# The Kind name for the ClusterTrainingRuntime.
+CLUSTER_TRAINING_RUNTIME_KIND = "ClusterTrainingRuntime"
+
+# The plural for the ClusterTrainingRuntime.
+CLUSTER_TRAINING_RUNTIME_PLURAL = "clustertrainingruntimes"
+
+# The label key to identify training phase where TrainingRuntime should be used.
+# For example, runtime for the pre-training or post-training.
+PHASE_KEY = "training.kubeflow.org/phase"
+
+# The value indicates that runtime can be used for the model pre-training.
+PHASE_PRE_TRAINING = "pre-training"
+
+# The value indicates that runtime can be used for the model post-training.
+PHASE_POST_TRAINING = "post-training"
+
+# The label key to identify the device (e.g. GPU, TPU) that is used for training.
+# TODO: Potentially, we should get this data from the Node selectors.
+DEVICE_KEY = "training.kubeflow.org/device"
+
+# Unknown indicates that the value can't be identified.
+UNKNOWN = "Unknown"
+
+# The label for CPU device in the container resources.
+CPU_DEVICE_LABEL = "cpu"
+
+# The default type for CPU.
+CPU_DEVICE_TYPE = "cpu"
+
+# The label for GPU device in the container resources.
+GPU_DEVICE_LABEL = "nvidia.com/gpu"
```

Comment:
The generic AMD accelerator label is `amd.com/gpu`. It might be more extensible to call this `NVIDIA_GPU_LABEL`.

Reply:
Yeah, it's a good point. let me rename this label to NVIDIA_GPU_LABEL